packages/expo-image-picker/ios/EXImagePicker/EXImagePicker.m

```diff
@@ -556,20 +556,29 @@ - (BOOL)hasCameraRollPermission
 
 - (NSString *)mapPreset:(NSString *)preset API_AVAILABLE(ios(11));
 {
-  if ([preset isEqualToString:@"Passthrough"]) return AVAssetExportPresetPassthrough;
-  if ([preset isEqualToString:@"LowQuality"]) return AVAssetExportPresetLowQuality;
-  if ([preset isEqualToString:@"MediumQuality"]) return AVAssetExportPresetMediumQuality;
-  if ([preset isEqualToString:@"HighestQuality"]) return AVAssetExportPresetHighestQuality;
-  if ([preset isEqualToString:@"H_264_640x480"]) return AVAssetExportPreset640x480;
-  if ([preset isEqualToString:@"H_264_960x540"]) return AVAssetExportPreset960x540;
-  if ([preset isEqualToString:@"H_264_1280x720"]) return AVAssetExportPreset1280x720;
-  if ([preset isEqualToString:@"H_264_1920x1080"]) return AVAssetExportPreset1920x1080;
-  if ([preset isEqualToString:@"H_264_3840x2160"]) return AVAssetExportPreset3840x2160;
-  if ([preset isEqualToString:@"H_264_960x540"]) return AVAssetExportPreset960x540;
-  if ([preset isEqualToString:@"H_264_960x540"]) return AVAssetExportPreset960x540;
-  if ([preset isEqualToString:@"HEVC_1920x1080"]) return AVAssetExportPresetHEVC1920x1080;
-  if ([preset isEqualToString:@"HEVC_3840x2160"]) return AVAssetExportPresetHEVC3840x2160;
-  return AVAssetExportPresetPassthrough;
+  static NSDictionary* presetsMap = nil;
+  if (!presetsMap) {
```

Comment:
I think we don't need to check whether `presetsMap` is set. Static variables are initialized only once, so it should be safe to just define them as a dictionary from the beginning.

Reply:
I've done this to avoid compilation error (**"Initializer element is not a compile-time constant"**).
you can read more about this [here](https://stackoverflow.com/questions/22534251/static-nsdictionary-const-lettervalues-in-a-method-does-not-compil).